home-work-app/src/app/components/modals/ContextMenu/StyledMenuItem.styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled, { StyledComponent }  from 'styled-components';
+import { stylesConfig } from '../../../configs/style.configs';
+
+export const StyledMenuItem: StyledComponent<'li', {}> = styled.li.attrs({
+    role: 'menuItem'
```

Comment:
If you will rework this component with styled `className` prop, you could pass this attribute as an other prop.
`<li className={className} role="menuItem">...</li>`

Reply:
Reworked